meson.build

```diff
@@ -0,0 +1,77 @@
+# init
+project('PilotDataSyncPlugin', 'cpp', default_options: ['cpp_std=c++20'], meson_version: '1.7.0')
+
+# imports
+fs = import('fs')
+ 
+# download sdk if not present
+# TODO
+
+# these defines are necessary as required by XPlane
+compiler_args = [
+  '-DXPLM200',
+  '-DXPLM210',
+  '-DXPLM300',
+  '-DXPLM301',
+  '-DXPLM303',
+  '-DIBM=1',
+]
+
+# TODO review if this is still necessary
+add_project_arguments('-Llib/SDK/Libraries/Win', language: 'cpp')
+
+# includes
+inc_dir = include_directories('include', 'lib/SDK/CHeaders/Widgets/', 'lib/SDK/CHeaders/XPLM/')
+
+compiler = meson.get_compiler('cpp')
+
+# slightly hacky method of retrieving the required absolute path to the precomp
+# libs until I find a better way
+lib_files = files('lib/SDK/Libraries/Win/XPLM_64.lib')
+lib_path = lib_files[0].full_path().replace('XPLM_64.lib', '')
+message(lib_path)
```

Comment:
I think you want:

```meson
lib_path = meson.current_source_dir() / 'lib/SDK/Libraries/Win'
```
?

Potentially relevant as well: https://mesonbuild.com/Shipping-prebuilt-binaries-as-wraps.html#shipping-prebuilt-binaries-as-wraps

because...